package.json

```diff
@@ -1,89 +1,48 @@
 {
-  "name": "tudelft-spacefinder",
-  "version": "1.0.0",
-  "description": "App to easily find available spaces on the TU Delft Campus.",
-  "author": "De Voorhoede (https://www.voorhoede.nl)",
   "private": true,
   "scripts": {
-    "analyze": "nuxt build --analyze",
-    "prebuild": "touch ./.env && rimraf dist/ && npm run data",
-    "build": "nuxt generate --fail-on-error",
-    "predev": "rimraf ./.nuxt && npm run data",
     "data": "node scripts/data/index.js",
-    "dev": "PORT=3463 nuxt",
-    "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
-    "mock:exchange": "node ./scripts/mock/exchange.js",
-    "precommit": "npm run lint",
-    "proxy": "ngrok http 3463",
-    "start": "http-server './dist/client/' -c-1 -o -p 3464",
+    "build": "nuxt build",
+    "dev": "nuxt dev",
+    "generate": "nuxt generate",
+    "preview": "nuxt preview",
+    "postinstall": "npm run data && nuxt prepare",
```

Comment:
I think this should be a `pre(build|dev|...)` script since data can change after install.
In newer projects we try to use this pattern:

```
{
  "prep": "run-s prep:*",
  "prep:data": "node scripts/data/index.js",
  "prep:nuxt": "nuxt prepare",
  "prep:...": "...",
  "predev": "npm run prep",
  "dev": "...",
  "prebuild": "npm run prep",
  "build": "..."
}
```

Reply:
nuxt prepare in our case fails if the data files are absent, but that's probably fixable and doesn't matter much, we just don't do `run-s`, ok. But I guess we need nuxt prepare on postinstall to make the dev environment happy. So we would either need to run the prep on both postinstall and pre(build|...), or be OK with the fact that we need to do npm run build locally to get the types, or be OK with the fact that we need to run rpm run data locally if the data changes. We can also have this piece of duplication probably

```
"postinstall": "npm run data && nuxt prepare",
"build": "npm run data && nuxt build",
"dev": "npm run data && nuxt dev",
```

I personally would re-fetch data locally manually (which means leaving everything like it is now) just to speed up the startup speed